express-basic/server.js

```diff
@@ -22,6 +22,10 @@ app.use('/color/:name', function(req, res) {
   
   if('red' === name) {
     code = '#FF0000'
```

Comment:
let's use lower case for all the colors as per your changes

Reply:
Ok, this is updated.